cmd/get/taskconfig_flags.go

```diff
@@ -41,8 +50,8 @@ func (TaskConfig) mustMarshalJSON(v json.Marshaler) string {
 // flags is json-name.json-sub-name... etc.
 func (cfg TaskConfig) GetPFlagSet(prefix string) *pflag.FlagSet {
 	cmdFlags := pflag.NewFlagSet("TaskConfig", pflag.ExitOnError)
-	cmdFlags.StringVar(&(taskConfig.ExecFile),fmt.Sprintf("%v%v", prefix, "execFile"), taskConfig.ExecFile, "execution file name to be used for generating execution spec of a single task.")
-	cmdFlags.StringVar(&(taskConfig.Version),fmt.Sprintf("%v%v", prefix, "version"), taskConfig.Version, "version of the task to be fetched.")
-	cmdFlags.BoolVar(&(taskConfig.Latest),fmt.Sprintf("%v%v", prefix, "latest"), taskConfig.Latest, "flag to indicate to fetch the latest version, version flag will be ignored in this case")
+	cmdFlags.String(fmt.Sprintf("%v%v", prefix, "execFile"), *new(string), "execution file name to be used for generating execution spec of a single task.")
```

Comment:
Haythams chage for default value binding is not in yet , so until then we will have to maintain these files manually.
We will need to Var version of the function

Can you please test workflows you have touched to make sure they are working fine after your changes